backend/ttrs/serializers.py

```diff
@@ -7,6 +7,7 @@
 
 from .models import Student, College, Department, Major, Course, Lecture, Evaluation, TimeTable
 
+from .recommend import recommend
```

Comment:
If you do not need `RecommendSerializer`, this import is not necessary

Reply:
Removed corresponding line